packages/experimental/src/components/UserProfileImage/UserProfileImage-test.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright IBM Corp. 2020, 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { TestScheduler } from 'jest';
+import { expPrefix } from '../../global/js/settings';
+
+import React from 'react';
+
+import { UserProfileImage } from '.';
+
+// const {
+//   defaultProps: { backgroundColor, icon, size, theme },
+//   name,
+// } = UserProfileImage;
+
+const { backgroundColor, icon, size, theme } = UserProfileImage;
+
+jest.setTimeout(15000);
```

Comment:
You might not need this now, as we've bumped up the timeout for all the tests globally (to 10000, actually, but we can make it higher if needed).